<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Draw3dParametersControlWidget</class> 
 <widget class="QWidget" name="Draw3dParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draw3dParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>style</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="styleComboBox">
       <property name="toolTip">
         <string>style</string>
       </property>
       <property name="currentIndex">
         <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>Points</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Textured</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Point Color</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="RgbColorParametersControlWidget" name="pointColorControlWidget">
       <property name="toolTip">
         <string>Point Color</string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Point Color Override</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="pointColorOverrideCheckBox">
       <property name="toolTip">
         <string>Point Color Override</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>Point Size</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="pointSizeSpinBox">
       <property name="toolTip">
         <string>Point Size</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>Edge Color</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="RgbColorParametersControlWidget" name="edgeColorControlWidget">
       <property name="toolTip">
         <string>Edge Color</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>Edge Color Override</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="edgeColorOverrideCheckBox">
       <property name="toolTip">
         <string>Edge Color Override</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>Edge Width</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="edgeWidthSpinBox">
       <property name="toolTip">
         <string>Edge Width</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>Face Color</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="RgbColorParametersControlWidget" name="faceColorControlWidget">
       <property name="toolTip">
         <string>Face Color</string>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>Face Color Override</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="faceColorOverrideCheckBox">
       <property name="toolTip">
         <string>Face Color Override</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
       <property name="text">
         <string>Show caption</string>
       </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="showCaptionCheckBox">
       <property name="toolTip">
         <string>Show caption</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
       <property name="text">
         <string>Font Size</string>
       </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="fontSizeSpinBox">
       <property name="toolTip">
         <string>Font Size</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
       <property name="text">
         <string>Font Width</string>
       </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="fontWidthSpinBox">
       <property name="toolTip">
         <string>Font Width</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
       <property name="text">
         <string>Font Color</string>
       </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="RgbColorParametersControlWidget" name="fontColorControlWidget">
       <property name="toolTip">
         <string>Font Color</string>
       </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_13">
       <property name="text">
         <string>Texture Corrodinates</string>
       </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QComboBox" name="textureCorrodinatesComboBox">
       <property name="toolTip">
         <string>Texture Corrodinates</string>
       </property>
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>No Texture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto Decide</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Force Texture</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_14">
       <property name="text">
         <string>Texture Alpha</string>
       </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSpinBox" name="textureAlphaSpinBox">
       <property name="toolTip">
         <string>Texture Alpha</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_15">
       <property name="text">
         <string>Texture Scale</string>
       </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QDoubleSpinBox" name="textureScaleSpinBox">
       <property name="toolTip">
         <string>Texture Scale</string>
       </property>
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>2000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_16">
       <property name="text">
         <string>Decal Matrix Type</string>
       </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QSpinBox" name="decalMatrixTypeSpinBox">
       <property name="toolTip">
         <string>Decal Matrix Type</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>19</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>11</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_17">
       <property name="text">
         <string>Decal Left Cam</string>
       </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QCheckBox" name="decalLeftCamCheckBox">
       <property name="toolTip">
         <string>Decal Left Cam</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_18">
       <property name="text">
         <string>Decal Left Alpha</string>
       </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QSpinBox" name="decalLeftAlphaSpinBox">
       <property name="toolTip">
         <string>Decal Left Alpha</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_19">
       <property name="text">
         <string>Decal Right Cam</string>
       </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QCheckBox" name="decalRightCamCheckBox">
       <property name="toolTip">
         <string>Decal Right Cam</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_20">
       <property name="text">
         <string>Decal Right Alpha</string>
       </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QSpinBox" name="decalRightAlphaSpinBox">
       <property name="toolTip">
         <string>Decal Right Alpha</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
   <item row="21" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="21" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>styleComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pointColorControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pointColorOverrideCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pointSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>edgeColorControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>edgeColorOverrideCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>edgeWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>faceColorControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>faceColorOverrideCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>showCaptionCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>fontSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>fontWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>fontColorControlWidget</sender>
   <signal>paramsChanged()</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>textureCorrodinatesComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>textureAlphaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>textureScaleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>decalMatrixTypeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>decalLeftCamCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>decalLeftAlphaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>decalRightCamCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>decalRightAlphaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Draw3dParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
  <customwidget>
   <class>RgbColorParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>corestructs/coreWidgets/rgbColorParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RgbColorParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>corestructs/coreWidgets/rgbColorParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RgbColorParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>corestructs/coreWidgets/rgbColorParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RgbColorParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>corestructs/coreWidgets/rgbColorParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
</ui>
